src/views/AccountsListing/AccountsListing.js

```diff
@@ -41,7 +42,7 @@ import {
 } from '../../components/Accounts';
 
 import { refundClaimReturned } from '../../constants';
-import { localizeCurrencyAmount } from '../../components/util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../components/util/localizeCurrencyAmount';
```

Comment:
after updating `src/hooks/index.js` file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import